src/ios/CDVCamera.m

```diff
@@ -525,8 +525,24 @@ - (void)resultForImage:(CDVPictureOptions*)options info:(NSDictionary*)info comp
 
 - (CDVPluginResult*)resultForVideo:(NSDictionary*)info
 {
-    NSString* moviePath = [[info objectForKey:UIImagePickerControllerMediaURL] absoluteString];
-    return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:moviePath];
+    NSURL *url = [info objectForKey:UIImagePickerControllerMediaURL];
+
+    NSData *urlData = [NSData dataWithContentsOfURL:url];
+    NSArray   *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+    NSString  *documentsDirectory = [paths objectAtIndex:0];
+    NSString  *newfilePath = [NSString stringWithFormat:@"%@/%@", documentsDirectory, [url lastPathComponent]];
+
+     BOOL fileExists = [[NSFileManager defaultManager] fileExistsAtPath:newfilePath];
```

Comment:
This seems to be indented one space further than the block above it. Can you re-align all of the code in this function?